Tools/AP_Periph/README.md

```diff
@@ -44,7 +44,7 @@ An AP_Periph UAVCAN firmware supports these UAVCAN features:
  - parameter storage in flash
  - easy bootloader update
  - high resiliance features using watchdog, CRC and board checks
- - firmware update via MissionPlanner or uavcan-gui-tool
+ - firmware update via MissionPlanner or UAVCAN GUI tool when attached to an autopilot, or via USB, if provided on the device, using SLCAN
```

Comment:
- It's now called the DroneCAN GUI Tool :)
- This line could be interpreted as the autopilot updating a AP_Periph which is not yet supported. 
- Firmware update via USB has been available from almost the very beginning, it doesn't use SLCAN.
-

Reply:
better now?